src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -275,6 +275,8 @@
     short-summary: The location to store the deployment metadata.
   - name: --name -n
     short-summary: The deployment name.
+  - name: --what-if-result-format
+    short-summary: The format of What-If results. Applicable when --confirm-with-what-if is set.
```

Comment:
`--what-if-result-format` the name of this parameter looks long, could you add a short alias?

Reply:
Create an alias `-r`. Also added it to the `what-if` commands for the `--result-format` parameter to keep it consistent.